Plot/src/main/java/io/deephaven/plot/util/tables/ColumnHandlerFactory.java

```diff
@@ -399,6 +400,20 @@ public double getDouble(int i) {
                     return value == null ? Double.NaN : DateTimeUtils.epochNanos(value);
                 }
 
+            };
+        } else if (type.equals(ZonedDateTime.class)) {
+            return new ColumnHandlerHandle(tableHandle, columnName, type, plotInfo) {
+                @Override
+                public TypeClassification typeClassification() {
+                    return TypeClassification.TIME;
+                }
+
+                @Override
+                public double getDouble(int i) {
+                    final Instant value = DateTimeUtils.toInstant((ZonedDateTime) getDataColumn().get(i));
+                    return value == null ? Double.NaN : DateTimeUtils.epochNanos(value);
+                }
```

Comment:
Does this even get used? I have the impression that web is snapshotting data directly rather than via any Plot code. Which is good, because this is a pretty sub-optimal way to fetch bulk data.

Reply:
Here is some test code I have been using.

```python
from deephaven import time_table
from deephaven.time import to_j_time_zone
from deephaven.plot.figure import Figure

tz = to_j_time_zone("ET")
t = time_table("PT00:00:01").update(["X=toZonedDateTime(Timestamp,tz)","Y=i"])
m = t.meta_table
p = Figure().plot_xy(series_name="Test", t=t, x="X", y="Y").show()
```

In the 0.28.1 demo, this fails with:

```
java.lang.UnsupportedOperationException: Unsupported numeric data type: columnName=X type=class java.time.ZonedDateTime
```

The equivalent Groovy is passing in my branch code.
```groovy

tz = timeZone("ET")
t = timeTable("PT00:00:01").update("X=toZonedDateTime(Timestamp,tz)","Y=i")
m = t.meta()
p = figure().plot_xy("Test", t, "X", "Y").show()
```

I will disable and test the code again.

Reply:
Confirmed that commenting out the new code results in an exception: ```java.lang.UnsupportedOperationException: Unsupported numeric data type: columnName=X type=class java.time.ZonedDateTime```

Reply:
A deep dive indicates that some, but not all of this code is called.  The methods to get type are called, but the `getDouble` methods are not called.  Removal of those functions should be part of a more comprehensive refactor and not this PR.